extension/popup/popupView.jsx

```diff
@@ -508,60 +508,73 @@ this.popupView = (function() {
     );
   };
 
-  const SearchResultsContent = ({
-    search,
-    cardImage,
-    displayText,
-    onSearchImageClick,
-    setMinPopupSize,
-  }) => {
-    if (!search) return null;
-
-    const { searchResults, index, searchUrl } = search;
-    const card = cardImage;
-    const next = searchResults[index + 1];
-    const cardStyles = card ? { height: card.height, width: card.width } : {};
-    const imgAlt =
-      card && card.alt
-        ? `Click to show search results: ${card.alt}`
-        : "Show search results";
-
-    if (card) {
-      setMinPopupSize(card.width, card.height);
+  class SearchResultsContent extends PureComponent {
+    constructor(props) {
+      super(props);
+
+      this.state = {
+        showBanner: true,
+      };
+    }
+
+    componentDidMount() {
+      setTimeout(() => {
+        this.setState({ showBanner: false });
+      }, 4000);
     }
 
-    const onSearchCardClick = () => {
-      onSearchImageClick(searchUrl);
+    onSearchCardClick = event => {
+      this.props.onSearchImageClick(this.props.search.searchUrl);
     };
 
-    return (
-      <React.Fragment>
-        <TextDisplay displayText={displayText} />
-        <div id="search-results">
-          {/* FIXME: img alt is using "next" title, but it should use the card title, which is not available in the data  */}
-          {card ? (
-            <img
-              id="search-image"
-              alt={imgAlt}
-              onClick={onSearchCardClick}
-              style={cardStyles}
-              src={card.src}
-              role="button"
-            />
-          ) : null}
-          {next ? (
+    render() {
+      if (!this.props.search) return null;
+
+      const { searchResults, index, searchUrl } = this.props.search;
+      const card = this.props.cardImage;
+      const next = searchResults[index + 1];
+      const cardStyles = card ? { height: card.height, width: card.width } : {};
+      const imgAlt =
+        card && card.alt
+          ? `Click to show search results: ${card.alt}`
+          : "Show search results";
+
+      if (card) {
+        this.props.setMinPopupSize(card.width);
+      }
+
+      return (
+        <React.Fragment>
+          <TextDisplay displayText={this.props.displayText} />
+          <div id="search-results">
+            {/* FIXME: img alt is using "next" title, but it should use the card title, which is not available in the data  */}
```

Comment:
Oops, I fixed this issue but forgot to remove the comment...

... but not really related to this PR